src/cli/domain/init-template/index.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+const path = require('path');
+const scaffold = require('./scaffold');
+const installTemplate = require('./install-template');
+const createComponentDir = require('./create-component-dir');
+const initPackage = require('./init-package');
+const utils = require('./utils');
+
+module.exports = function(options, callback) {
+  const {
+    componentName,
+    templateType,
+    logger = console,
+    cli = 'npm'
+  } = options;
+  const componentPath = path.join(process.cwd(), componentName);
```

Comment:
It actually happen in one place. The other calls to `process-cwd()` are for different matters (local compiler, resolve for path being passed for template). But agree on hoisting it up in the facade. 👍